.github/workflows/ci.yml

```diff
@@ -51,23 +51,18 @@ jobs:
           docgen
           tar -zcf docs.tar.gz docs
 
-      - name: Create release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish release
+        uses: goreleaser/goreleaser-action@v5
+        if: ${{ !contains(github.ref, '-') }}
         with:
-          tag_name: ${{ github.ref }}
-          release_name: ${{ github.ref }}
-          draft: false
-          prerelease: false
-
-      - name: Upload docs asset
-        uses: actions/upload-release-asset@v1
+          args: release --clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish non-main release
```

Comment:
we don't need this... this is when you need to make a binary for a branch of something.. there's no goflow binary